R/predict.R

```diff
@@ -0,0 +1,512 @@
+#' Predict Method for RtGam Models
+#'
+#' Generates predictions from an `RtGam` fit. Prediction dates can be specified
+#' flexibly using multiple approaches.
+#'
+#' @param object An `RtGam` object from [RtGam()].
+#' @param parameter A string specifying the prediction parameter. Options are
+#'   `"obs_cases"`, `"incidence"`, `"r"`, and `"Rt"`. Defaults to `"obs_cases"`.
+#' @param horizon Optional. Integer specifying forecast days from the last date
+#'   in the fit. For example, `horizon = 7` returns a 7-day forecast.
+#' @param min_date,max_date Optional. A date-like object. See details for more
+#'   information on specification.
+#' @param n Number of posterior samples to use. Default is 100.
+#' @param mean_delay Optional. Numeric mean delay used in the prediction.
```

Comment:
This isn't hugely informative as a doc string. From not having looked at the code I assume you are shifting Rt back using this

Reply:
https://github.com/CDCgov/cfa-gam-rt/pull/48/commits/6b6ec2a73f2c8f5fc37c5664a89a71aa349ddee7